public/index.html

```diff
@@ -1,33 +1,30 @@
 <html>
 
 <head>
-  <title>A-Frame Social VR Demo</title>
-  <script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/socket.io/1.4.5/socket.io.min.js"></script>
-  
-  <script src="easyrtc/easyrtc.js"></script>
-  <script src="./app.bundle.js"></script>
+    <title>A-Frame Social VR Demo</title>
+    <script src="https://aframe.io/releases/0.7.0/aframe.min.js"></script>
+    <script src="./app.bundle.js"></script>
 </head>
 
 <body>
-  <a-scene
-    networked-scene="adapter: easyrtc;
+    <a-scene networked-scene="adapter: janus;
+                     room: 2;
+                     serverURL: wss://quander.me:8989;
                      audio: true;
                      debug: true;
-                     connectOnLoad: false"
-    mute-mic="eventSrc: a-scene; toggleEvents: action_mute">
-    <a-assets>
-      <img id="grid" src="assets/grid.png" crossorigin="anonymous">
-      <img id="sky" src="https://cdn.aframe.io/360-image-gallery-boilerplate/img/sechelt.jpg" crossorigin="anonymous" />
+                     connectOnLoad: false" mute-mic="eventSrc: a-scene; toggleEvents: action_mute">
```

Comment:
Minor, but prefer to keep attributes broken up like they were before.

Reply:
More than minor, IMO. I find long lines of attributes pretty unreadable, especially for A-Frame.